indico/modules/rb/operations/admin.py

```diff
@@ -62,3 +63,45 @@ def update_room(room, args):
         update_principals_permissions(room, current, acl_entries)
     _populate_room(room, args)
     db.session.flush()
+
+
+def create_area(area_data):
+    bounds = area_data['bounds']
+    top, bottom = bounds['north_east'], bounds['south_west']
+    is_default = area_data.get('default', False)
+
+    if is_default:
+        MapArea.query.update({MapArea.is_default: False}, synchronize_session='fetch')
+
+    new_area = MapArea()
```

Comment:
It's a choice of style, but wouldn't it be more readable to pass those as `kwargs` of the constructor?